src/sentry/utils/eventuser.py

```diff
@@ -125,6 +125,21 @@ def from_snuba(result: Mapping[str, Any]) -> EventUser:
             user_ident=result.get("user_id"),
         )
 
+    @classmethod
+    def for_tags(cls, project_id: int, values):
```

Comment:
This needs a test

Reply:
Oh I see, is this already tested by tests/sentry/data_export/processors/test_issues_by_tag.py since you're importing the new class?